lib/matplotlib/dates.py

```diff
@@ -325,8 +325,14 @@ def __init__(self, fmt):
         self.fmt = fmt
 
     def __call__(self, s):
-        """s : string to be converted
-           return value: a date2num float
+        """
+        Parameters
+        ----------
+        s : str
+
+        Returns
+        -------
+        num : date2num float
```

Comment:
I would not invent a name for the return value here (per https://numpydoc.readthedocs.io/en/latest/format.html you can just specify the type without a name).